src/dataset/mod.rs

```diff
@@ -83,6 +84,7 @@ pub(crate) fn serialize_data<X: RealNumber, Y: RealNumber>(
     Ok(())
 }
 
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Why is this disabled for wasm32?.

Reply:
no "datasets" module and its content for now in the wasm binary, it will just take space.

Reply:
I think that for the wasm binary we could just compile without `datasets` feature to solve that